crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -863,44 +863,44 @@
             "name": "Result",
             "value": [
               {
-                "object_id": "0xa14c6b812b94fe613c5bcebb5eeb1d449e251616",
+                "objectId": "0xa14c6b812b94fe613c5bcebb5eeb1d449e251616",
```

Comment:
Note for future generations (and @patrickkuo who it looks like owns this file): this looks like a breaking change, but it's actually not--openrpc.json was wrong previously. The problem examples.rs uses `ObjectInfo` to construct example outputs, whereas our endpoints actually return `SuiObjectInfo`. The former type uses snake case and the latter uses camel case (hence the changes here).